content/sigs/cloud-native/index.adoc

```diff
@@ -16,36 +16,24 @@ tags:
 - configuration-as-code
 - jcasc
 leads:
-- "carlossg"
-- "markyjackson-taulia"
+- "marckk"
 participants:
```

Comment:
TBH I see no point in listing participants. I was rather -0.5 for that from very beginning when @bitwiseman introduced JEP-4, and I still do not see a need in that. We just end up with always-outdated lists on SIG pages. Regular participants are visible in the mailing list, chats and meeting notes. Should be enough IMHO

Reply:
I understand your point; however, every single other SIG lists participants. So we'll keep this for now, to be consistent with all the other SIGs.